src/OrbitGl/TimeGraphLayout.cpp

```diff
@@ -18,7 +18,7 @@ TimeGraphLayout::TimeGraphLayout() {
   space_between_gpu_depths_ = 2.f;
   space_between_tracks_ = 10.f;
   space_between_tracks_and_thread_ = 5.f;
-  space_between_gpu_subtracks_ = -5.f;
+  space_between_subtracks_ = -5.f;
```

Comment:
I also have no idea why this can be negative.

Reply:
Probably Florian. I'm in of making it positive in this PR, shouldn't be too many changes, @florian-kuebler  what do you think??

Reply:
To be honest, I don't remember why this is negative. I am all for making it positive (but I think we need to adjust some other values here, as otherwise it would not look nice). I would argue to not do this in this PR.

Reply:
Added a bug http://b/193018071 for this. And we will address this in another PR.